modin/core/storage_formats/base/query_compiler.py

```diff
@@ -3452,6 +3452,17 @@ def groupby_unique(
             series_groupby=True,
         )
 
+    def groupby_ohlc(
+        self,
+        by,
+        axis,
+        groupby_kwargs,
+        agg_args,
+        agg_kwargs,
+        is_df,
+    ):
+        return DataFrameDefault.register(lambda df: df.ohlc())(self)
```

Comment:
this isn't using any of arguments to the function... why is that?

Reply:
When defaulting to pandas, we don't need any of these arguments. These are for other storage formats that won't default to pandas!